src/breadcrumb-group/item/item.tsx

```diff
@@ -28,11 +28,12 @@ export function BreadcrumbItem<T extends BreadcrumbGroupProps.Item>({
     <div className={clsx(styles.breadcrumb, isLast && styles.last)}>
       <a
         {...focusVisible}
-        href={item.href || '#'}
+        href={isLast ? undefined : item.href || '#'}
         className={clsx(styles.anchor, { [styles.compressed]: isCompressed })}
         aria-current={isLast ? 'page' : undefined} // Active breadcrumb item is implemented according to WAI-ARIA 1.1
         aria-disabled={isLast && 'true'}
         onClick={isLast ? preventDefault : onClickHandler}
+        tabIndex={isLast ? 0 : undefined} // tabIndex is added to the last crumb to keep it in the index without an href
```

Comment:
Isn't this logically equivalent to this since the default tabIndex for a non-disabled anchor tag would be based on the dom order?

```
tabIndex={0}
```

Reply:
You are asking why not just have tabIndex = 0 on all the links because it would be functionally equivalent? Is that correct?

It certainly would be functionally equivalent. I was favoring not adding unnecessary/redundant attributes and since isLast is already evaluated I don't believe it would have much if any performance impact to make it conditional, but I'm happy to take either approach, especially if one if more idiomatic to Cloudscape

Reply:
I have no strong preference here.